### PR TITLE
Add Njalla

### PR DESCRIPTION
**Pull request by jtagcat**

### CODE REVIEW

File: _data/domains.yml [hunk 1]
```diff
@@ -397,6 +397,17 @@ websites:
...
 
+    - name: Njalla
+      url: https://www.networksolutions.com/
+      twitter: njal_la
+      email_address: support@njal.la
+      img: njalla.png
+      tfa: Yes
+      software: Yes
+
+      img: networksolutions.png
```
Comment (Carlgo11) [inline _data/domains.yml]:
```suggestion
```

File: _data/domains.yml [hunk 2]
```diff
@@ -397,6 +397,17 @@ websites:
...
+      tfa: No
```
Comment (Carlgo11) [inline _data/domains.yml]:
```suggestion
```

File: _data/domains.yml [hunk 3]
```diff
@@ -397,6 +397,17 @@ websites:
       img: networksolutions.png
       tfa: No
 
+    - name: Njalla
+      url: https://www.networksolutions.com/
```
Comment (Carlgo11) [inline _data/domains.yml]:
```suggestion
      url: https://njal.la/
```

File: _data/domains.yml [hunk 4]
```diff
@@ -397,6 +397,17 @@ websites:
...
+      twitter: njal_la
```
Comment (Carlgo11) [inline _data/domains.yml]:
```suggestion
```

File: _data/domains.yml [hunk 5]
```diff
@@ -397,6 +397,17 @@ websites:
...
+      email_address: support@njal.la
+      img: njalla.png
+      tfa: Yes
+      software: Yes
+
```
Comment (Carlgo11) [inline _data/domains.yml:405]:
```suggestion
```

File: _data/domains.yml [hunk 6]
```diff
@@ -397,6 +397,17 @@ websites:
       img: networksolutions.png
       tfa: No
 
+    - name: Njalla
+      url: https://www.networksolutions.com/
+      twitter: njal_la
+      email_address: support@njal.la
```
Comment (Carlgo11) [inline _data/domains.yml]:
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Social media handles and email address isn't needed if the site supports 2FA. See [CONTRIBUTING.MD](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#new-sites) for the different tags and when to use them.

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Oh I just noticed the site is ranked above 200k on [alexa](https://www.alexa.com/siteinfo/njal.la) 😞 
We've set a rule on only allowing new sites that are ranked below 200k on Alexa as a way to keep our lists more manageable. Sadly that means we can't approve this pull request.
If the site reaches a rank below 200k, feel free to request to have this PR reopened.

Again, sorry for the inconvenience!

**Comment by jtagcat** [comment]
Sorry, I tought Njall.la is a good addition and didn't check.

**Pull request closed without merging**